### DISCUSSION TITLE
Slimproto doesn't work?

### DISCUSSION THREAD

**Discussion by lukaszzyla** [music-assistant/support#2515] [Q&A]
Hi!
I have LMS 8.5.2 configured and working. I have 3 sources available as players using squeezelite  https://sourceforge.net/projects/lmsclients/files/squeezelite/linux/squeezelite-2.0.0.1488-x86_64.tar.gz/download
My LMS is on separate host and configured with 3 inputs as players:
![image](https://github.com/music-assistant/hass-music-assistant/assets/36972413/3a1fbe85-0d14-4d54-ab33-fd65ede7779c)

I have 3 players available in squeezelite, which show up in HA with squeezebox integration, but they are not visible in MA. I can add them to my through HA integration as a HA players - but this seems like not solving this directly.
Anyone has an idea how to fix it?

**Comment by OzGav** [comment]
Is it a Network problem? Do you have any other Squeezelite devices that do show up? If not install one on your PC to test.
